.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: "weekly"
    groups:
      actions:
        patterns:
          - "*"
    labels:
      - "github-actions"
      - "dependabot"
```

If these are grouped then it will just be a single PR per week, at most.